src/app/features/cinematic/index.js

```
@@ -0,0 +1,278 @@
+      })
+      .addCase(getCinematicByCategory.fulfilled, (state, action) => {
+        state.isVideoFetching = false;
+      })
+      .addCase(getCinematicByCategory.rejected, (state, action) => {
+        state.isVideoFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(cinematicTopVideo.pending, (state) => {
+        state.isTopVideoFetching = true;
+      })
+      .addCase(cinematicTopVideo.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)